tests/test_benchmark_scenarios.py

```diff
@@ -71,6 +78,7 @@ def setup_method(self):
     def test_benchmark_AB_grid_pv(self, margs):
         r"""
         Benchmark test for simple case grid connected PV. Since the PV is already installed, this tests makes sure that the PV generation is totally used to supply the load and the rest in take from the grid.
```

Comment:
Do you agree to that sentence? I think that @ursulaelmir only tested that the energy balance is fine.
I maybe we should change it:
```suggestion
        Benchmark test for simple case grid connected PV, in which a fix capacity of PV is installed (installedCap, no optimization). 
        Assertions performed:
        - The sum of energy consumption from the grid and PV generation is equal to the load (and flow to excess sink) at all times (ie. energy balance)
```

Reply:
check done in 5af2d6c 
applied your suggestions in 4fd7659